colour/io/tests/test_ies_tm2714.py

```diff
@@ -239,9 +237,7 @@ def test_write(self):
         """
 
         sd_r = SpectralDistribution_IESTM2714(
-            os.path.join(RESOURCES_DIRECTORY, 'Fluorescent.spdx'))
-
-        sd_r.read()
+            os.path.join(RESOURCES_DIRECTORY, 'Fluorescent.spdx')).read()
 
         sd_r.path = os.path.join(self._temporary_directory, 'Fluorescent.spdx')
         self.assertTrue(sd_r.write())
```

Comment:
@BPearlstine : I think you need to modify the next line to `sd_t = SpectralDistribution_IESTM2714(sd_r.path).read()` and you are good! 👍